PyCon2017Namibia/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for PyCon2017Namibia project.
+
+Generated by 'django-admin startproject' using Django 1.10.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.10/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.10/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '=i(x(0rk(y7a0r%^w5*51ypr%cfyn#38@!#i8114(3k1!wix&d'
```

Comment:
@euclidian move secret in other file (like secret.py) and add ignore for secret.py
see here : https://github.com/kartoza/docker-mezzanine/tree/develop/django_project/core/settings